pymc_marketing/mmm/data_generator.py

```diff
@@ -0,0 +1,611 @@
+import warnings
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+
+from pymc_marketing.mmm.transformers import (
+    delayed_adstock,
+    geometric_adstock,
+    logistic_saturation,
+)
+
+warnings.filterwarnings("ignore", category=FutureWarning)
+FIGSIZE = (15, 8)
```

Comment:
Matplotlib / Seaborn / Arviz use a global variable (something rc) that controls things like this. We should let those be used by default, and users can tweak them that way as well.